### PR TITLE
[@types/onfleet_node-onfleet] Fix mismatched type for verifyKey() 

### PR DESCRIPTION
**Pull request by Miikis**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)

### CODE REVIEW

File: types/onfleet__node-onfleet/onfleet__node-onfleet-tests.ts [hunk 1]
```diff
@@ -1,82 +1,79 @@
 import Onfleet = require('@onfleet/node-onfleet');
 
 const onfleet = new Onfleet('test-api-key');
-const isValid = onfleet.verifyKey();
+const isValid = onfleet.verifyKey().then().catch();
```
Comment (marcobeltempo) [inline types/onfleet__node-onfleet/onfleet__node-onfleet-tests.ts:4]: For consistency, you can do `const isValid = await onfleet.verifyKey();`

File: types/onfleet__node-onfleet/Resources/Administrators.d.ts [hunk 2]
```diff
@@ -1,44 +1,44 @@
 import { OnfleetMetadata } from '../metadata';
 
 declare class Admin {
-  create(obj: Admin.CreateAdminProps): Promise<Admin.OnfleetAdmin>;
```
Comment (marcobeltempo) [inline types/onfleet__node-onfleet/Resources/Administrators.d.ts:4]: Let's leave out the formating changes in this PR. We could introduce a style config for the entire project separately.

**Review by marcobeltempo — CHANGES_REQUESTED** [review]
Nice catch on the `verifyKey` return type! I left a couple of comments regarding formatting.

File: types/onfleet__node-onfleet/onfleet__node-onfleet-tests.ts (hunk 1 above)
Comment (Miikis) [reply types/onfleet__node-onfleet/onfleet__node-onfleet-tests.ts:4]: I would do this if I could but JS doesn't allow you to do `await` in the global scope — always has to be inside of an `async` function.

File: types/onfleet__node-onfleet/Resources/Administrators.d.ts (hunk 2 above)
Comment (Miikis) [reply types/onfleet__node-onfleet/Resources/Administrators.d.ts]: 😎 removed the prettier stuff.

**Review by marcobeltempo — APPROVED** [review]

**Comment by Miikis** [comment]
Ready to merge

**Pull request merged**